### PR TITLE
Add a merging of update events after the relation handling

### PR DESCRIPTION
**Pull request by reiterl**
Merges all Update Events with `fields` into one. 
It doesn't merge the update events with `list_fields`.

### CODE REVIEW

File: openslides_backend/action/action.py [hunk 1]
```diff
@@ -397,10 +397,48 @@ def process_write_requests(
...
+
+        result: List[Event] = []
+        for fqid in events_by_fqid:
+            result.extend(self.merge_update_events_helper(events_by_fqid[fqid]))
+
+        return result
+
+    def merge_update_events_helper(self, events: List[Event]) -> List[Event]:
+        if len(events) < 2:
+            return events
```
Comment (jsangmeister) [inline openslides_backend/action/action.py:423]: I think the algorithm also works for one element so you can remove this check, right?

File: openslides_backend/action/action.py [hunk 2]
```diff
@@ -397,10 +397,48 @@ def process_write_requests(
...
+        for event in update_events:
+            events_by_fqid[event["fqid"]].append(event)
+
+        result: List[Event] = []
+        for fqid in events_by_fqid:
+            result.extend(self.merge_update_events_helper(events_by_fqid[fqid]))
+
+        return result
+
+    def merge_update_events_helper(self, events: List[Event]) -> List[Event]:
```
Comment (jsangmeister) [inline openslides_backend/action/action.py:421]:
```suggestion
    def merge_update_events_for_fqid(self, events: List[Event]) -> List[Event]:
```
helper is always very generic

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Looks good! Tested it with the performance test from https://github.com/OpenSlides/openslides-backend/pull/1247 and the time used for 100 votes was reduced from about 1.9s to 1.1s and the amount of events from ~900 to ~200.

File: openslides_backend/action/action.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/action.py:423]: Removed, the code doesn't need it.

File: openslides_backend/action/action.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/action.py:421]: Done. Renamed.

**Review by jsangmeister — APPROVED** [review]
In the long run, we can look to streamline the relation handling, so this is no longer necessary, but works good for now.

**Pull request merged**